tests/integ/agent/test_end_to_end.py

```diff
@@ -40,7 +41,7 @@ async def test_end_to_end_tcp_multiple_flushes():
     @metric_scope
     async def do_work(metrics):
         metrics.put_dimensions({"Operation": "Agent"})
-        metrics.put_metric(metric_name, 100, "Milliseconds")
+        metrics.put_metric(metric_name, 100, "Milliseconds", StorageResolution.HIGH)
```

Comment:
Lets remove High. Its not required for libraries as we are not checking it against 1 sec internal in the response

Reply:
Done!